### PR TITLE
sample-app/cleanup.sh: fix usage of not-installed killall command

### PR DESCRIPTION
**Pull request by php-coder**
PTAL @rhcarvalho @mfojtik @kargakis

### CODE REVIEW

**Comment by rhcarvalho** [comment]
The next thing you'll run into with the cleanup script is unmounting volumes, see #2629.

My PR was rejected at the time, but I think it was a matter of miscommunication.

**Comment by kargakis** [comment]
LGTM [merge]

Thanks! @mfojtik fyi

File: examples/sample-app/cleanup.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-killall openshift
+pkill --exact openshift
```
Comment (mfojtik) [inline examples/sample-app/cleanup.sh]: @smarterclayton does this option exists on Mac? (I have bad feeling that you have just `-x`)

Comment (php-coder) [inline examples/sample-app/cleanup.sh]: Thanks, looks like you're right and Mac doesn't have a long version. I've corrected it.

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**